templates/index.html

```diff
@@ -40,15 +42,17 @@ <h1 class="d-flex justify-content-center">URL shortener</h1>
             type="text"
             id="lname"
             class="form-control"
-            name="longURL"
+            name="longUrl"
             required
           />
           <br />
-          <button type="submit" class="btn btn-primary mb-3">
+          <button type="submit" id="submit-btn" class="btn btn-primary mb-3">
             Shorten!!!!
           </button>
         </div>
       </fieldset>
     </form>
+    <div id="newURL"></div>
+    <script src="script.js"></script>
```

Comment:
remove `src="script.js"` and then just move then contents in `script.js` inside here. It will look something like:

```html
<script type="text/javascript">
function myfunction(event) {
  // content here...
}
</script>
```